framework/src/testing/create_transaction.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import { Transaction } from '@liskhq/lisk-chain';
+import { codec } from '@liskhq/lisk-codec';
+import { getAddressAndPublicKeyFromPassphrase } from '@liskhq/lisk-cryptography';
+import { signTransaction, validateTransaction } from '@liskhq/lisk-transactions';
+import { AssetClass } from './types';
+
+interface CreateTransactionInput {
+	moduleID: number;
+	assetClass: AssetClass;
+	asset: Record<string, unknown>;
+	nonce?: bigint;
+	fee?: bigint;
+	passphrase?: string;
+	networkIdentifier?: Buffer;
```

Comment:
networkIdentifier is required?

Reply:
It is only required if user wants to sign transaction. Since we are making that optional, it is not required.